### ISSUE TITLE
Orderstoinvoice.php

### ISSUE THREAD

**Issue by bobjml69** [Dolibarr/dolibarr#16724]
# Instructions
*This is a template to help you report good issues. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *replace the bracket enclosed texts with meaningful information*
- *remove any unused sub-section*

# Bug
[the function to invoice mass order is not implemented on the button "Invoices Orders"]

## Environment
- **Version**: [*V13.02*]
- **OS**: [*WINDOWS*]
- **Web server**: [*Webserver type and version*]
- **PHP**: [*7.4*]
- **Database**: [*MYSQLca*]
- **URL(s)**: [*Affected URL(s)*]

## Expected and actual behavior
[*Verbose description*]

## Steps to reproduce the behavior
[*Verbose description*]

## [Attached files](https://help.github.com/articles/issue-attachments) (Screenshots, screencasts, dolibarr.log, debugging informations…)
[*Files*]

**Comment by atm-maxime** [comment]
Could you please give more informations / screenshots and remove the template text in the issue ?

**Comment by bobjml69** [comment]
hello

in V12 , when you want to invoice for one customer more than 1 order, 
you can do it by clicking on one order then customer and you get this 
bar

on v12 the link to "invoice orders " is :
https://erp.alpes-images.com/commande/orderstoinvoice.php?socid=7919 
<https://orderstoinvoice.php?socid=7919>

in V13 the same button, the link is :
https://erp.alpes-images.com/commande/list.php?socid=7919 
<https://list.php?socid=7919>

the result is différent as with we cannot get this page and separate 
each order with bloc


BR

JM LIGNER

------ Message d'origine ------
De: "Maxime Kohlhaas" ***@***.***>
À: "Dolibarr/dolibarr" ***@***.***>
Cc : "bobjml69" ***@***.***>; "Author" 
***@***.***>
Envoyé : 16/03/2021 22:08:49
Objet : Re: [Dolibarr/dolibarr] Orderstoinvoice.php (#16724)

**Comment by atm-maxime** [comment]
Hi. I just checked this. Yes the old "ordertoinvoice" screen has been replaced by the standardized order list with the mass action "bill orders". For me it does exactly the same and gives more possibilities (no need to do invoicing one customer at a time for example). Only difference I see is that in the old screen, order references were copied in the public note.
How did you "separate each order with bloc" before ?
